doc/api/cli.md

```diff
@@ -1229,6 +1229,16 @@ Starts the Node.js command line test runner. This flag cannot be combined with
 See the documentation on [running tests from the command line][]
 for more details.
 
+### `--test-coverage`
```

Comment:
I'm not a huge fan of adding more and more (unscoped) options that are specific to the test runner or watch mode. (There's a discussion about something like Python's `-m` somewhere, I am sure.)

In particular, if this is specific to the test runner, it could be scoped. On the other hand, users who dislike `node:test` might want to use a native coverage report option without having to use `node:test`.

Reply:
> On the other hand, users who dislike node:test might want to use a native coverage report option without having to use node:test.

This change is specifically about supporting code coverage in `node:test`, because I believe it's a feature that test runners should support. Code coverage in Node itself has been supported for years. I would encourage such users to use the coverage tool in their test runner of choice or something like c8. They could also PR the changes to Node themselves, but that seems unlikely since coverage has already been in core for years.